src/components/builder/Items/Armors/ArmorsStyle.js

```diff
@@ -0,0 +1,98 @@
+import { colors } from "../../../../utils/colors";
+import { StyleSheet } from "react-native";
+
+export const styles = StyleSheet.create({
+    globalItemContainer: {
+      position: 'relative',
+      zIndex: 100,
+      width: '100%',
+      height: '100%',
+      top: 0,
+      left: 0,
+    },
+    itemNavBar: {
+      justifyContent: "start"
+    },
+    closeButton: {
+      position: 'absolute',
+      top: 10, // Ajoutez la valeur que vous souhaitez
+      right: 10, // Ajoutez la valeur que vous souhaitez
+      color: colors.neutralBlackColor, // Remplacez par votre couleur
+      transition: '0.2s', // Cela peut ne pas être nécessaire en React Native
+      padding: 8,
+    },
+    searchBar: {
+        width: '100%',
+        alignItems: 'flex-start',
+        justifyContent: 'flex-start',
+    },
+    input: {
+        width: '90%',
+        marginVertical: 8,
+    },
+    filters: {
+      // Style pour les filtres
+    },
+    filterDropdowns: {
+      // Style pour les dropdowns de filtre
+    },
+    filterDropdown: {
+      // Style pour un dropdown de filtre individuel
+    },
+    itemContainer: {
+        flexDirection: "row",
+        justifyContent: "center"
+    },
+    item: {
+        backgroundColor: colors.backgroundItem,
+        padding: 16, // Ajoutez la valeur que vous souhaitez
+        color: colors.neutralWhiteColor, // Ajoutez votre couleur de texte
+        cursor: 'pointer', // Pas nécessaire en React Native
```

Comment:
si pas nécessaire, supprime le, ainsi que tous les petits commentaires made in chatgpt